src/layouts/UserLayout.js

```diff
@@ -37,31 +39,35 @@ class UserLayout extends React.PureComponent {
     const { routerData, match } = this.props;
     return (
       <DocumentTitle title={this.getPageTitle()}>
-        <div className={styles.container}>
-          <div className={styles.top}>
-            <div className={styles.header}>
-              <Link to="/">
-                <img alt="logo" className={styles.logo} src={logo} />
-                <span className={styles.title}>Ant Design</span>
-              </Link>
+        <Layout className={styles.container}>
```

Comment:
I think the Layout doesn't need to be used here, because this is a very simple structure, just <div> is ok 😄

Reply:
i restored it